benchmark/utils/CxxTests/CxxStdlibPerformance.h

```diff
@@ -29,6 +44,18 @@ inline void initSet(size_t size) {
     }
 }
 
+#if !defined(__linux__)
```

Comment:
Could we use `#if __has_include(<span>)` instead of `!defined(__linux__)`? That would make the reasoning behind the guard slightly more clear, and it would be easier to run the benchmarks locally with newer Linuxes that have the `span` header installed.
(It's fine to leave the `#if os` check on the Swift side as is, since I don't think there is a better way to perform this check in Swift)

Reply:
Of course, thanks!